dali_tf_plugin/dali_dataset_op.cc

```diff
@@ -311,27 +427,11 @@ REGISTER_OP("DALIDataset")
   .Attr("prefetch_queue_depth: int")
   .Attr("cpu_prefetch_queue_depth: int")
   .Attr("gpu_prefetch_queue_depth: int")
-  .Attr("shapes: list(shape) >= 1")
-  .Attr("dtypes: list({half, float, uint8, int16, int32, int64}) >= 1")
+  .Attr("output_shapes: list(shape) >= 1")
```

Comment:
I think shape and dtype is more aligned with TF naming convention for arguments with that purpose.

Reply:
Not in the datasets. The use output_shapes and output_dtypes.
And I wanted to align the API with `from_generator` as it works in similar fashion but for DALI Pipelines instead of function:
https://www.tensorflow.org/api_docs/python/tf/data/Dataset#from_generator

I can revert it back, because it's an API change - OTOH I was thinking about adding a note that the Dataset is under development and the API can change between releases.
What do you think? 
@awolant was in favor of changing the API.

Reply:
sure